apps-rendering/src/image/srcsets.ts

```diff
@@ -27,7 +27,26 @@ interface Srcsets {
 
 // ----- Functions ----- //
 
-const getSubdomain = (domain: string): string => domain.split('.')[0];
+/**
+ * Get the resizing source for a given CAPI asset
+ *
+ * @param url The asset URL from CAPI
+ * @returns An image resizer source
+ */
+const getSource = (url: URL): string => {
+	const subdomain = url.hostname.split('.')[0];
```

Comment:
If you like that kind of stuff…

```suggestion
	const [subdomain] = url.hostname.split('.');
```